R/log_shiny_input_changes.R

```diff
@@ -44,9 +44,12 @@ log_shiny_input_changes <- function(
   stopifnot(is.character(namespace) && length(namespace) == 1)
   stopifnot(is.character(excluded_inputs))
   stopifnot(is.character(excluded_pattern) && length(excluded_pattern) == 1)
-  stopifnot(inherits(session, "session_proxy"))
+  stopifnot(inherits(session, "session_proxy") || inherits(session, "ShinySession"))
 
-  if (logger::TRACE > logger::as.loglevel(get_val("TEAL.LOG_LEVEL", "teal.log_level", "INFO"))) {
+  # Log even if written in lower case or numeric values
+  log_level <- get_val("TEAL.LOG_LEVEL", "teal.log_level", "INFO")
+  if (!is.numeric(log_level)) log_level <- toupper(log_level)
+  if (logger::TRACE > logger::as.loglevel(log_level)) {
```

Comment:
Unless the log_level is TRACE or higher than 700  it won't record anything

Reply:
Yeah, so it records only for DEBUG, ERROR, FATAL, WARN,

Reply:
But I don't get those numeric values

```r
> logger::as.loglevel(700L)
Error in as.loglevel.integer(700L) : 
  x %in% as.integer(loglevels) is not TRUE
```

Reply:
```r
> logger::as.loglevel(700)
Error in as.loglevel.numeric(700) : 
  x %in% as.integer(loglevels) is not TRUE
```

Reply:
You are right, `DEBUG` was too high. Sorry